salt/metalk8s/bootstrap/certs/clients.sls

```diff
@@ -0,0 +1,9 @@
+include:
+  - metalk8s.kubeadm.init.certs.apiserver
```

Comment:
This is not a 'client' cert, if that's what the state name refers to. Also applies to e.g. `sa`, `etcd-server` and maybe others.

Reply:
Name suggestion ?

Reply:
=> #823